### ISSUE TITLE
Training Details

### ISSUE THREAD

**Issue by 1211536289** [VITA-Group/TurbNet#3]
Can you describe some training details? For example, batch size, how much loss has decreased, and graphics card model.

**Comment by BBuniverse** [comment]
Could you provide the supplementary for those training details?

Thank you in advance.
Cheers

**Comment by GavinSun0921** [comment]
Could you provide the training code? Thanks.
